### ISSUE TITLE
findFontFamily() error for unitless line-height values

### ISSUE THREAD

**Issue by m-allanson** [stylelint/stylelint#1480]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where the `findFontFamily` util does not correctly identify a font. The bug is triggered by checking a string that includes a unitless line-height.

> Which rule, if any, is this issue related to?

The `findFontFamily` util

> What CSS is needed to reproduce this issue?

```css
.a {
  font: 16px/3 Arial;
}
```

> Which version of stylelint are you using?

6.6.0

> What did you expect to happen?

The font 'Arial' should be identified

> What actually happened (e.g. what warnings or errors you are getting)?

The font '3 Arial' was identified

I think this should be resolved before #1398
